api/v1beta2/common_types.go

```diff
@@ -60,6 +60,7 @@ type ClusterStorage struct {
 // RedisExporter interface will have the information for redis exporter related stuff
 type RedisExporter struct {
 	common.RedisExporter `json:",inline"`
+	SecurityContext      *corev1.SecurityContext `json:"securityContext,omitempty"`
```

Comment:
We should move this fields to common.RedisExporter

Reply:
Done.